lib/rules/declaration-block-no-redundant-longhand-properties/README.md

```diff
@@ -145,6 +145,18 @@ a {
 }
 ```
 
+You should use a longhand property to have only the specific value inherited. So, the following patterns are also not considered problems:
+
+<!-- prettier-ignore -->
+```css
+a {
+  margin-top: 1px;
+  margin-right: 2px;
+  margin-bottom: 3px;
+  margin-left: inherit;
+}
+```
+
```

Comment:
Let's remove this as our patterns don't tend to show edge cases. 

We can add something to do _extended description_ at the top instead. For example, expand the following line

> This rule will only complain if you've used the longhand equivalent of _all_ the properties that the shorthand will set.

To:

"This rule will only complain if you've used the longhand equivalent of _all_ the properties that the shorthand will set and if their values are not `inherit`."

Reply:
Thanks for review! I removed a description of edge case. https://github.com/stylelint/stylelint/pull/6419/commits/616a536d977f7a6c6ce283634a942737045fce77
Then, I updated the description to your suggestion. https://github.com/stylelint/stylelint/pull/6419/commits/f442a05438dcbcb783ee6c388f34dc801783e47a